docs/2_4_common_schema.adoc

```diff
@@ -736,7 +738,7 @@ latexmath:[\mathbf{T}_{shift} = ] `shiftCounter / resolution`.
 
 This value must be greater than 0.
 
-This attribute is allowed if <<supportsFraction,`supportsFraction = true`>> and is required if <<intervalVariability,intervalVariability == `constant`>> or <<intervalVariability,intervalVariability == `fixed`>>.
+This attribute is allowed if <<supportsFraction,`supportsFraction = true`>>.
```

Comment:
I removed that requirement completely because constant clocks do not need to have a shift necessarily.